Sources/NIOHTTP1/HTTPEncoder.swift

```diff
@@ -74,6 +74,9 @@ private func writeTrailers(wrapOutboundOut: (IOData) -> NIOAny, ctx: ChannelHand
     }
 }
 
+// starting about swift-5.0-DEVELOPMENT-SNAPSHOT-2019-01-20-a, this doesn't get automatically inlined, which costs
+// 2 extra allocations so we need to help the optimiser out.
+@inline(__always)
```

Comment:
@eeckstein between swift-5.0-DEVELOPMENT-SNAPSHOT-2019-01-13-a and swift-5.0-DEVELOPMENT-SNAPSHOT-2019-01-20-a this suddenly didn't get inlined anymore without any code changes.

Is that expected? It was stable from at least Swift 4.0.1 until now and then suddenly last week it broke...